files/angular.ui-router/update.json

```diff
@@ -1,8 +1,8 @@
 {
   "packageManager": "github",
   "name": "angular.ui-router",
-  "repo": "angular-ui/ui-router",
+  "repo": "angular-ui/ui-router/tree/legacy",
```

Comment:
`/tree/legacy` is not part of the repo name, its a path inside the `angular-ui/ui-router` repo. So this is incorrect